src/main/java/net/dv8tion/jda/api/entities/User.java

```diff
@@ -624,4 +636,123 @@ public static int getRaw(@Nonnull Collection<UserFlag> flags)
             return getRaw(flags.toArray(EMPTY_FLAGS));
         }
     }
+    
+    /**
+     * Represents the information about {@link User User}'s primary guild
+     */
+    class PrimaryGuild
+    {
+        private final String identityGuildId;
```

Comment:
We use `long` for every this everywhere in JDA. Let's just do `private final long guildId` and make `PrimaryGuild` implement `ISnowflake` with that id.